website/public/demo/demo.mjs

```diff
@@ -55,6 +55,83 @@ function synchronizeSize() {
   shadowCodeInputElement.style.height = codeInputElement.style.height;
 }
 
+function showErrorMessageBox(mark, posCursorX) {
+  const div = createErrorBox(
+    mark,
+    posCursorX,
+    mark.attributes["data-message"].value,
+    mark.attributes["data-code"].value,
+    mark.attributes["data-severity"].value
+  );
+  let body = document.querySelector("body");
+  body.appendChild(div);
+}
+
+function createErrorBox(
+  markedElement,
+  posCursorX,
+  errorMessage,
+  code,
+  severity
+) {
+  // TODO: Change background color based of the severity
+  let div = document.createElement("div");
+  const { bottom } = markedElement.getBoundingClientRect();
+  div.setAttribute("class", "error-message-box");
+  div.innerText = `${code} - ${errorMessage}`;
+  div.style.position = "fixed";
+  div.style.overflow = "auto";
+  div.style.top = `${Math.trunc(bottom)}px`;
+  div.style.left = `${posCursorX}px`;
+  div.classList.add("error-box");
```

Comment:
> Did you mean to add two different classes (error-message-box and error-box)? I see that JS uses one and CSS uses the other. Why not have one class?

Was a mistake, I changed the reference to this div to be id, instead of classes.